v2/tools/generator/internal/astbuilder/slices.go

```diff
@@ -24,6 +24,24 @@ func MakeSlice(listType dst.Expr, len dst.Expr) *dst.CallExpr {
 	}
 }
 
+// MakeAppendSlice returns the call expression for making a slice with a specified length that can be
+// appended to using append(...)
+//
+// make([]<value>, 0, <len>)
+func MakeAppendSlice(listType dst.Expr, len dst.Expr) *dst.CallExpr {
+	return &dst.CallExpr{
+		Fun: dst.NewIdent("make"),
+		Args: []dst.Expr{
+			listType,
+			&dst.BasicLit{
+				Value: "0",
+				Kind:  token.INT,
+			},
+			len,
+		},
+	}
```

Comment:
Working too hard.

```suggestion
	return CallFunc(
		"make",
		listType,
		IntLiteral(0),
		len)
	}
```

Reply:
You might want to improve `MakeSlice()` too.

Reply:
Fixed both